mailbox/plugin/deleted-messages-vault/src/main/java/org/apache/james/vault/blob/BlobStoreDeletedMessageVault.java

```diff
@@ -21,6 +21,8 @@
 
```

Comment:
If you rebase on my work this commit is no longer required and should be discarded.